src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,72 @@
-import React from 'react';
+import React, { useState } from 'react';
+import Classnames from 'classnames';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { ResponseError } from '../../types/ReponseError';
+import { MovieCard } from '../MovieCard';
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+enum Response {
+  title = 'Title',
+  error = 'Error',
+}
+
+type Props = {
+  addMovie: (movie:Movie) => void
+};
+
+export const FindMovie: React.FC<Props> = ({ addMovie }) => {
+  const [search, setSearch] = useState<string>('');
+  const [answear, setAnswear] = useState<Movie | {} | ResponseError>({});
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [error, setError] = useState<boolean>(false);
```

Comment:
it's extra
if you put boolean like initial value - your typescript already know about this type
```suggestion
  const [search, setSearch] = useState('');
  const [answear, setAnswear] = useState<Movie | {} | ResponseError>({});
  const [isLoading, setIsLoading] = useState(false);
  const [error, setError] = useState(false);
```